content/docs/manuals/networking-internals/proxy-conversion.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Proxy Conversion"
+summary: ""
+draft: false
+weight: 315
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+In order to make the service layer networking functional, a worker-level tun-proxy is transparently instantiated
+as part of the Oakestra network component. The following picture is an example of what happens in a worker node
+based on the IPv4 implementation of the NetManager component. It is worth noting, that
+**IPv4 and IPv6 work identically**. For simplicity, we will stick to IPv4 addresses.
+
+## Example
+
+In this example, we have two worker nodes, namely Node 1 and Node 2, each containing two containers.
+The containers are instantiated and managed by the NodeEngine (See High-Level Architecture wiki), while
```

Comment:
Link to be added

Reply:
Removed the cross reference, as it does not provide any more information.